docs/hydra/guides/oauth2-webhooks.mdx

```diff
@@ -0,0 +1,295 @@
+---
+id: claims-at-refresh
+title: OAuth2 webhooks
+sidebar_label: OAuth2 webhooks
+keywords:
+  - webhooks
+  - custom
+  - customize
+  - claims
+  - jwt
+  - access
+  - token
+---
+
+# Customizing claims with OAuth2 webhooks
+
+```mdx-code-block
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem'
+```
+
+You can modify aspects of the OpenID Connect and access tokens returned from Hydra's OAuth2 token endpoint. A typical use case is
+adding custom claims to the tokens issued by Hydra.
+
+To customize the token response, you must configure a webhook endpoint in your OAuth2 configuration. Before the token is issued to
```

Comment:
```suggestion
To customize the token response configure a webhook endpoint in your OAuth2 configuration. Before the token is issued to
```

Reply:
This sentence is now definitely missing a comma :D

Reply:
rephrased